src/Components/Blazor/Build/src/targets/Blazor.MonoRuntime.targets

```diff
@@ -21,7 +21,7 @@
 
   <Target
     Name="_BlazorCopyFilesToOutputDirectory"
-    DependsOnTargets="PrepareBlazorOutputs"
+    DependsOnTargets="PrepareBlazorOutputs;$(_BlazorCopyFilesToOutputDirectoryDependsOn)"
```

Comment:
This is a bit unorthodox, but it will go away once we move to use static web assets.